client/src/redux/actions/authAction.js

```diff
@@ -0,0 +1,32 @@
+import {SIGN_IN, SIGN_OUT, SIGN_IN_SUCCESS, SIGN_IN_FAILURE} from "./types";
+
```

Comment:
@Gold-Turtle ,

Nice, remember what we spoke about just now on discord. Research redux-thunk & axios. Axios is how we will communicate with our server. Redux-Thunk allows us to return functions from actions. The returned function will execute, make the axios call, and then return us the object [type & payload] to be processed to our reducer.